src/styleUtils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$primary-text: #313237;
+$secondary-text: #89939a;
+$icons: #b4bdc4;
+$elements: #e2e6e9;
+$hover: #fafbfc;
+$green: #27ae60;
```

Comment:
good practice is to declare variables for colors like `$color-green`, `$color-red` etc.

don't use no descriptive names like `icons` or `elements` because they not pointing colors, rather something different.

if you have problems to name colors correctly you can try this [https://coolors.co/d1d1d1-dbdbdb-85c7f2-4a4855-4c4c4c](url) here you can paste you Hex values and it will generates for you helpful name :)